tubemq-server/src/main/java/org/apache/tubemq/server/broker/metadata/TopicMetadata.java

```diff
@@ -37,7 +37,7 @@
     private int unflushThreshold = 1000;
     // data will be flushed to disk when unflushed message count exceed this.
     private int unflushInterval = 10000;
-    @Deprecated
+    // data will be flushed to disk when unflushed data size reach this threshold.
     private int unflushDataHold = 10000;
```

Comment:
The initial value of unflushdatahold is set to 0. By default, this function is turned off. When it is online, it is turned on as needed.